### ISSUE TITLE
cocoapods doesn't work

### ISSUE THREAD

**Issue by Gargo** [teodorpatras/SideMenuController#23]
[!] Unable to find a specification for `SideMenuController (~> 0.1.3)`

**Comment by ahsas-sharma** [comment]
I faced the same issue as well but is now resolved. After some research on StackOverflow, turned out there was a problem with my cocoapods master repo. Enter the following commands in terminal:
```
pod repo remove master
```
Followed by - 
```
pod setup --verbose
```
The second operation could take a while to complete due to a 300MB+ sized download. Just gotta be patient with it. After it completes, just run pod install and it should work like a charm. 

Here's a link to the SO page (check out the 2nd answer with 16 upvotes) - 
http://stackoverflow.com/questions/25913733/unable-to-find-a-specification-in-cocoapods
